javascript-data-model/order-history.js

```diff
@@ -0,0 +1,63 @@
+// const { PureComponent } = require("react");
+
+var orderHistory = [
+  {
+    order_placed: { month: 'August', day: 4, year: 2020 },
+    total: 34.00,
+    ship_to: 'JS Masher',
+    order_number: '114-3941689-8772232',
+    delivery_date: { delivered: 'Delivered', month: 'Aug', day: 8, year: 2020 },
+    order: [{
+      product_name: 'Javascript for impatient programmers',
+      author: 'Rauschmayer, Dr. Axel',
+      return_window: 'Return window closed on Sep 7, 2020',
+      price: 31.55
+    }]
+  },
+  {
+    order_placed: { month: 'July', day: 19, year: 2020 },
+    total: 44.53,
+    ship_to: 'JS Masher',
+    order_number: '113-9984268-1280257',
+    delivery_date: { delivered: 'Delivered', month: 'Jul', day: 20, year: 2020 },
+    delivery_message: 'Your package was delivered. It was handed directly to a resident.',
+    order: [{
+      product_name: 'The Timeless Way of Building',
+      author: 'Alexander, Christopher',
+      return_window: 'Return window closed on Aug 19, 2020',
+      price: 41.33
+    }]
+  },
+  {
+    order_placed: { month: 'July', day: 4, year: 2020 },
+    total: 17.22,
+    ship_to: 'JS Masher',
+    order_number: '114-2875557-9059409',
+    delivery_date: { delivered: 'Delivered', month: 'Jul', day: 7, year: 2020 },
+    delivery_message: 'Your package was delivered. It was handed directly to a resident.',
```

Comment:
You should totally have this property on each order for consistency, but `null` would be a good value for the orders that have no message

Reply:
added and changed to null

Reply:
Why do i keep it here, but remove the "author" from the arrays?

Reply:
@ameliatalkington Good question! That's because every order would potentially try to use this message, but not all items would even possibly have an author, so we would only provide that data on items that would have it, such as books. To make it even clearer, you might want to add a property for each item stating what type of item it is.